optics-core/src/Optics/Prism.hs

```diff
@@ -20,14 +20,21 @@ module Optics.Prism
   -- specialise types to obtain:
   --
   -- @
-  -- 'Optics.AffineFold.preview' :: 'Prism' s t a b -> s -> Maybe a
-  -- 'Optics.Review.review'  :: 'Prism' s t a b -> b -> t
+  -- 'Optics.AffineFold.preview' :: 'Prism'' s a -> s -> Maybe a
+  -- 'Optics.Review.review'  :: 'Prism'' s a -> a -> s
```

Comment:
I'm quite sure `::` should be aligned

Reply:
They are aligned in the rendered output (because the module prefix isn't displayed).

Reply:
I see, thanks for the explanation.